circle.yml

```diff
@@ -173,6 +173,9 @@ jobs:
     working_directory: ~/TheBrain2.0
     steps:
       - checkout
+      - run:
+          name: Generating branch name
```

Comment:
Name `branch name` is misleading, I suggest to use something like `build version label`